### ISSUE TITLE
No Dockerfile

### ISSUE THREAD

**Issue by khabdrick** [moja-global/FLINT.Cloud#58]
![2022-02-08_15-37](https://user-images.githubusercontent.com/57852546/153010180-ce2e24e0-d21c-4dec-abd8-7f75ee1a105c.png)

I tried to run the `docker build -t flint-api .` command to build the project locally, and I got the error above.

**Comment by chicken-biryani** [comment]
Hey, @khabdrick can please change the path to the directory that contains the docker files.

**Comment by khabdrick** [comment]
@chicken-biryani  thanks for the help, it works now. I didn't know the Dockerfile was in anothe location. Maybe it should be stated in the README, what do you think?

**Comment by khabdrick** [comment]
@chicken-biryani  thanks for the help, it works now. I didn't know the Dockerfile was in another location. Maybe it should be stated in the README, what do you think?

**Comment by chicken-biryani** [comment]
@khabdrick Have you followed the instructions given in the Readme while setting up the project on your local machine?

**Comment by khabdrick** [comment]
@chicken-biryani  Yes I did, I just didn't know the Dockerfile was in another directory other than the root.

**Comment by chicken-biryani** [comment]
@khabdrick then it should definitely be added in the Readme. We don't want others to face the same problem. Thanks for pointing it out. Can you open an issue followed by a pull request that solves the problem?

Thanks!
